maven-model-builder/src/test/java/org/apache/maven/model/interpolation/StringSearchModelInterpolatorTest.java

```diff
@@ -33,15 +33,18 @@
 import java.util.concurrent.Future;
 import java.util.concurrent.FutureTask;
 
+import static org.apache.commons.lang3.reflect.FieldUtils.readField;
```

Comment:
That's a problem because we want to remove Commons Lang from Core.

Reply:
Yes, I forgot about that, now I removed that class usage.